models/job.go

```diff
@@ -6,7 +6,8 @@ import (
 
 //Job represents a job metadata model and json representation for API
 type Job struct {
-	ID                           string    `json:"id"`
+	ID                           string    `bson:"_id" json:"id"`
+	CollectionID                 string    `bson:"collection_id,omitempty" json:"collection_id,omitempty"`
```

Comment:
I'd add bson values for the other fields in the struct too. Not strictly necessary if they match the field names but it helps with consistence should any refactoring take place.

Reply:
I've added the bson values and also removed the CollectionID attribute (I must have added this early on when I was unsure how to use mongo)!